content/articles/django-custom-template-tags-and-filters/index.md

```diff
@@ -0,0 +1,224 @@
+
+### Introduction.
+We are going to cover how to create Django custom template tags and filters.Django template language comes with built-in template tags such as `{% if %}` and filters such as `safe`.You may however find that  you require a functionality not included in django.You create them using python and avail them using the `load ` tag.Let’s get started! 
```

Comment:
Capitalize the words Django and Python throughout the article.

Reply:
Done
:rocket: